include/picongpu/particles/creation/creation.kernel

```diff
@@ -153,26 +147,26 @@ namespace picongpu
                         particleCreatorCtx[idx] = particleCreator;
 
                         // init particle creator functor for each virtual worker
-                        particleCreatorCtx[idx].init(acc, blockCell, linearIdx, localCellIndex);
+                        particleCreatorCtx[idx].init(acc, blockCell, idx, localCellIndex);
                     });
 
                     /* Declare counter in shared memory that will later tell the current fill level or
                      * occupation of the newly created target frames.
                      */
                     PMACC_SMEM(acc, newFrameFillLvl, int);
 
-                    ForEachIdx<IdxConfig<2, numWorkers>> onlyMasters{workerIdx};
+                    auto onlyMasters = lockstep::makeMaster(workerIdx);
```

Comment:
BUG!! This is not a single master we use two master worker to maintain out frame double buffer